client/packages/inventory/src/Stocktake/DetailView/SidePanel.tsx

```diff
@@ -20,16 +20,29 @@ import {
   useNavigate,
   RouteBuilder,
   UNDEFINED_STRING_VALUE,
+  BufferedTextInput,
 } from '@openmsupply-client/common';
 import { useStocktake } from '../api';
 import { canDeleteStocktake } from '../../utils';
 
 const AdditionalInfoSection: FC = () => {
   const t = useTranslation();
 
-  const { comment, user, createdDatetime, update } =
-    useStocktake.document.fields(['comment', 'user', 'createdDatetime']);
+  const { comment, user, createdDatetime, update, verifiedBy, countedBy } =
+    useStocktake.document.fields([
+      'comment',
+      'user',
+      'createdDatetime',
+      'countedBy',
+      'verifiedBy',
+    ]);
   const [bufferedComment, setBufferedComment] = useBufferState(comment ?? '');
+  const [bufferedCountedBy, setBufferedCountedBy] = useBufferState(
+    countedBy ?? ''
+  );
+  const [bufferedVerifiedBy, setBufferedVerifiedBy] = useBufferState(
+    verifiedBy ?? ''
+  );
```

Comment:
Oof such buffer states... one day we'll get new hooks everywhere 😭

Reply:
Would be great ;-;